### ISSUE TITLE
Remove from this org

### ISSUE THREAD

**Issue by xivk** [anyways-open/NetTopologySuite.IO.VectorTiles#19]
Remove from this org and migrate any issues that may still be relevant.

**Comment by HarelM** [comment]
Can you elaborate on this? Do you mean to move to NTS org?

**Comment by xivk** [comment]
Yes, this code has moved here: https://github.com/NetTopologySuite/NetTopologySuite.IO.VectorTiles

After a year it's probably fine to close this repo ;)

**Comment by HarelM** [comment]
Ahh, great to know! Thanks!
